apps/server/src/routes/v1/riders.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+import { registerRyder } from "../../controllers/riderControllers";
+import { getRiders } from "../../controllers/riderControllers/getRiderDetails";
```

Comment:
Then, clearing line 3 and importing the `getRiders` function into line 2. It helps to get the code to be more organized.
What do you think?